### PR TITLE
Add Vantiv to Payments section

### PR DESCRIPTION
**Pull request by oliver-donorschoose**
Vantiv is one of the biggest payment processors on the planet. Every year, they process >20B billion transactions and $726B in volume. Unfortunately, their merchant-facing console/portal does not offer 2FA.

### CODE REVIEW

File: _data/payments.yml [hunk 1]
```diff
@@ -135,6 +135,12 @@ websites:
       software: Yes
       doc: https://support.stripe.com/questions/how-do-i-enable-two-step-verification
 
+    - name: Vantiv
+      url: https://www.vantiv.com/
+      twitter: Vantiv
+      img: vantiv.png
+      tfa: No
+
```
Comment (Jawshy) [inline _data/payments.yml]: Please remove these blank spaces.

**Review by Jawshy — CHANGES_REQUESTED** [review]
Thanks for submitting a pull request, @oliver-donorschoose. 

Could you please add their Facebook page? 
```ruby
      facebook: Vantiv
```

[Travis CI](https://travis-ci.org/2factorauth/twofactorauth/jobs/182438994): 
```
1. img/payments/vantiv.png is not 32x32 pixels.
2. img/payments/vantiv.png should not be larger than 2500 bytes. It is currently 24786 bytes.
```

Would you also be able to edit the image so that it is both 32 by 32 pixels and under 2,500 bytes? You can use tinypng.com to compress the image once it has been resized.

**Comment by oliver-donorschoose** [comment]
@Jawshy Thanks for the feedback, we should be good to go now! Can you approve?

**Review by stephengroat — APPROVED** [review]

**Comment by Jawshy** [comment]
That looks better. Thanks again, @oliver-donorschoose. I have decided to also update the icon to the same one that their website uses as a favicon.

**Review by Jawshy — APPROVED** [review]

**Comment by oliver-donorschoose** [comment]
Thanks @Jawshy! Although I don't see the addition at https://twofactorauth.org/ > Payments... Let me know if there's a final step I need to take.

**Review by mxxcon — APPROVED** [review]

**Comment by Jawshy** [comment]
@oliver-donorschoose, there are two steps to go after approval. They are merging into the `master` branch (done just a few minutes ago) and then merging into the `gh-pages` branch at which point the changes will go live. See [FAQ_Merge](https://github.com/2factorauth/twofactorauth/wiki/FAQ_Merge) for more details on the process.

**Pull request merged**